msal/src/main/res/raw/msal_native_auth_default_config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "broker_redirect_uri_registered": false,
```

Comment:
Interesting. I think this is needed to be **true** for proper redirect uri validation.

Reply:
I have made the change. @SammyO  FYI

Reply:
Hold on @SaurabhMSFT. Redirect URI should be optional, so wouldn't making this true make it in effect also required? @shahzaibj

Reply:
@SammyO That depends on how you evaluate these things when you do your own validation. For Android MSAL, redirect URI is always required. 

This particular parameter is just checking if redirect URI is of the correct format that support brokered auth.

Reply:
@SaurabhMSFT  in that case you can't simply make the change. We'll have to evaluate whether we can or not, as redirect URI should stay optional.

Reply:
I have reverted it to false.